frontend/www/js/omegaup/components/homepage/Slide.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="container-lg p-4">
```

Comment:
```suggestion
  <div class="container-lg p-4 d-print-none d-md-none d-lg-block">
```

usar [estas clases de Bootstrap 4](https://getbootstrap.com/docs/4.4/utilities/display/#hiding-elements) evitan tener L18-L24!

Reply:
Esas clases hacen que el componente se oculte hasta estar en `lg`. Las líneas L18-L24 hacen que el componente se despliegue con tamaño automático hasta llegar a `lg` donde tiene un tamaño predefinido de 500px (para ocupar la mayor parte del alto del viewport).

No encontré una clase tipo: `h-breakpoint-tamañoEnPixeles` para poder hacerlo directamente con bs4.